modules/apps/feature-flag/feature-flag-web/src/main/resources/META-INF/resources/js/FeatureFlagToggle.tsx

```diff
@@ -30,6 +38,18 @@ const FeatureFlagToggle = ({
 	const [disabled, setDisabled] = useState(initialDisabled);
 	const [toggled, setToggled] = useState(initialToggled);
 
+	FeatureFlagToggleRegistry.registerToggle(
```

Comment:
Well I think we are reinventing here what we can do with React.js. I personally don't really like this type of approach because we are using React.js the wrong way as an isolated component and a separate tree from the other components, ideally using this when we created it was a way to allow teams to slowly move their application to React.js instead of using this just as an isolated UI component this does not bring the real use of React.js.

I would encourage here to move the toggle list to React.js and do this in React.js, this should be simple to implement and we don't need to move the MT to React.js we can keep it in the taglib if we want the SSR, remember this [MT taglib is a React.js component](https://github.com/liferay-peds/liferay-portal/blob/274e744500f2f915f6d4b5b5b81363e4d8a52c4e/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ManagementToolbar.js) then we could call it in our application if we want.

Reply:
Thanks @matuzalemsteles! 
We will follow  your suggestion to use Clay management bar taglib